### ISSUE TITLE
Feature Request: Prompt to Always Load When Defining a New Layer Style, Brush, Brush Style, etc

### ISSUE THREAD

**Issue by camoody1** [photopea/photopea#5300]
The title is pretty self explanatory. Since Photopea is browser based and you can't easily save custom settings like those I listed, I think it would be great if you would prompt the user if he/she would like to load their newly defined item every time Photopea is opened (ie. Save to Local Storage).

9 times out of 10, when I create a new brush, layer style, brush style or font style (assuming you add those), I want thatvitem to be available to me in future Photopea sessions. If you could ask the user immediately upon "defining" the item if it should be saved for future sessions, you could then ask the user to name the item, save that item as an export, and immediately "import" it into the Local Storage option to always Load. This would save the user the awkward workaround of having to export the item, rename the file (from the default filename you assign it) and then importing the item just so they can select to always Load it from Local Storage.

**Comment by photopea** [comment]
I dont think we should promote using Edit - Local Storage as a reliable storage for brushes, fonts, etc. It is better to save these files to your hard drive.

Many people use browser extensions, which delete website data, and the Local Storage is deleted, too. Or if you want to move to a different browser, you would have to download them as a ZIP to your computer and load into a different browser.

**Comment by MichalSZZ** [comment]
Maybe these user files can be stored in cloud and related with user account?

Then user can have them anywhere on every computer and browser after log in to the account. It can be 50-100 MB storage space for non pro users.

That is only an idea.

**Comment by camoody1** [comment]
> Maybe these user files can be stored in cloud and related with user account?
> 
> Then user can have them anywhere on every computer and browser after log in to the account. It can be 50-100 MB storage space for non pro users.
> 
> That is only an idea.

Nice idea for leveraging a user's Photopea storage.

**Comment by MichalSZZ** [comment]
In adobe CC i have something like libraries and I have some assets in cloud and I have acces to them in any of programs from the Adobe family. I can share these assets with my team. They can see the name of the library and assets inside. I can update it  every time and even better (when they use cloud smart object - it will be always up to date when user want to update it).

But now we are talking about only a litle part of cloud storage for user data.

**Comment by photopea** [comment]
We have 5 million users. With 100 MB per user, we would need 500 TB of cloud storage, we can not afford that.

**Comment by camoody1** [comment]
> We have 5 million users. With 100 MB per user, we would need 500 TB of cloud storage, we can not afford that.

Understandable. What about taking advantage of the user's own One Drive or Google Drive? Maybe save the items to a specifically named folder. If the folder is missing or empty, nothing gets loaded.

I know this is getting pretty deep at this point. Just trying to think of ways to allow Photopea to act a little more like a regular application with user specific customizations.

**Comment by photopea** [comment]
I will come up with a solution.
